algebird-core/src/main/scala/com/twitter/algebird/Aggregator.scala

```diff
@@ -103,6 +103,16 @@ object Aggregator extends java.io.Serializable {
   def sortedReverseTake[T: Ordering](count: Int): MonoidAggregator[T, PriorityQueue[T], Seq[T]] =
     new mutable.PriorityQueueToListAggregator[T](count)(implicitly[Ordering[T]].reverse)
   /**
+   * Immutable version of sortedTake
```

Comment:
can we add a comment as to when you might want to use this?

A few years ago I benchmarked the PriorityQueue version to be much faster (but that was before we had the benchmark subproject sadly). Given that users generally don't care what the middle type is, when would I want this?  I assume when I am disassembling the Aggregator and running each part myself (as happens in scalding or spark), and for some reason I don't want mutable intermediate objects (I can't think of a reason we care about that if it is a lot faster though).